modules/integration/tests-integration/tests-metadata/src/test/java/org/wso2/carbon/registry/metadata/test/wsdl/WSDLValidationTestCase.java

```diff
@@ -184,17 +211,9 @@ public void deleteResources()
         for (Service service : serviceManager.getAllServices()) {
             if (service.getQName().getLocalPart().equals("Info")) {
                 serviceManager.removeService(service.getId());
-            } else if (service.getQName().getLocalPart().equals("WeatherForecastService")) {
+            } else if (service.getQName().getLocalPart().equals("GeoIPService")) {
```

Comment:
Hi,
Thanks for the comment. Fix is done in pull request[1] commit[2].
[1] - https://github.com/wso2/product-greg/pull/71
[2] - https://github.com/prasa7/product-greg/commit/b45f2f321b13765dc123a64ae55014c9eb21474c